project/RELEASES.md

```diff
@@ -0,0 +1,168 @@
+# Scala LTS maintenance and release cycle
+
+## Scala LTS and Scala Next
+
+Since Scala 3.3.0, there are always two versions of Scala 3 being developed at the same time: Scala Next and Scala LTS.
```

Comment:
```suggestion
Since Scala 3.3.0, there are always two versions of Scala 3 being developed at the same time: Scala LTS (currently 3.3.x) and Scala Next (will be 3.4.x, 3.5.x, and so on, until a new LTS is designated). This scheme was previously outlined in [this August 2022 blog post](https://www.scala-lang.org/blog/2022/08/17/long-term-compatibility-plans.html).
```